scopes/pkg/pkg/pkg.docs.md

```diff
@@ -38,88 +43,127 @@ This extensions gets properties to add to the package.json in the variants confi
 
 #### Publish
 
-Configure the `publishConfig` prop with your registry data. For example:
+> Auto-publishing is triggered by the "export" process only when `publishConfig` or `name` are set.
```

Comment:
This is a bit confusing. a publish to external (not bit.dev) registry will happen only if publish config or name are set.
But if both are not set it is still available in the bit.dev registry (it's not done via the npm publish command, but this is just implementation details and not really interesting for the user).
So they do not really affect the publish but affect the target registry.